frontend/src/hooks/tests/useChat.spec.tsx

```diff
@@ -1,6 +1,6 @@
 import { act, renderHook } from '@testing-library/react';
 import { beforeEach, describe, expect, it, Mock, Mocked, vi } from 'vitest';
-import { MutableRefObject } from 'react';
```

Comment:
Note: `MutableRefObject` has been discontinued in React 19. SonarCloud is very unhappy about keeping this even though it is not currently a breaking change. This has been replaced with `RefObject`s. All other changes related to `MutableRefObject` are for the same reason.

Reply:
To confirm, there's no way to disambiguate between mutable and immutable ref objects?

Reply:
correct, they're all `refobject`s now and it is mutable